meetings/2023-02-15.md

```diff
@@ -45,6 +51,28 @@
+* eP: We have very clear use cases and implementations. CSS couples OIDC Provider (OP) and the Resource Server. I am using an authorization agent that needs to receive notifications so we have existing codebases with gaps that can give feedback to the spec work. 
+* SC: Do you think the PRotocol spec should be more clean on when to use Solid-OIDC? Because you mentioned it is social agent depends on a person. Whereas other cases (server-server), should we start mentioning these more clearly? Notifications as example: if notification server sends a message to a receiver, it is something we acknowledge for how some notification channels can be. Is that enough to raise it in PRotocol, Solid-OIDC for client-driven stuff, but if you want to do server-driven we don't have it nailed down yet but there is this other spec. Solid-OIDC or WebID-OIDC weren't the only way to authenticate, but only now we are trying to cover these major scenarios. 
+* eP: It'd be beneficial that spec clarifies Solid-OIDC is intended to be used by a user, possibly create an issue for server-side clients, and link to those as ongoing work. We should not only talk about authentication. We still miss similar AuthZ when using Solid-OIDC. We don't have a way for a user to authorize a client to access data. 
+* SC: Henry raised this in issues/discussions. The issued I linked (URL) there is one line: 
+  >Subscription Client lets the Notification Receiver know about the Notification Sender and their public key.
+  >Notification Receiver sets Authorization rules for Notification Sender.
+* SC: We need to allow receiver to add an authorization rule for who will have access to send a message. 
+* eP: That's valid in some cases. [Issue notifications#134](https://github.com/solid/notifications/issues/134). ACME should be able to switch from `sent.ex` to `hermes.ex` to deliver notifications In some cases, it is valid to allow change between services while the channel is active, without breaking it. There's a way to do that if you set policies on the receiver to ACME you can push as a claim a data/access grant, as request is being made you can provide proof that you're authorized. Inrupt does this with access grants, and we have work in progress. We need to clear those issues and define those patterns, probably follow what Inrupt is doing with access grants. This is all about access delegation which has various prior work. We need to find ways to all work together on this.
+* SC: If we can have this topic on the radar people can be aware that it's happening at least on Wed so people can join in. Otherwise we continue as we have been on issues/chat, but I don't find it too active. 
+* eP: Tomorrow we'll have discussion with Jackson. He proposed a different AuthN approach for webhook channel type so we will be able to dive deeper. Hopefully Aaron will join too. 
```

Comment:
```suggestion
* eP: That's valid in some cases. [Issue notifications#134](https://github.com/solid/notifications/issues/134). ACME should be able to switch from `sent.ex` to `hermes.ex` to deliver notifications. In some cases, it is valid to allow change between services while the channel is active, without breaking it. There's a way to do that: if you set policies on the receiver to ACME, you can push a data/access grant as a claim; as request is being made, you can provide proof that you're authorized. Inrupt does this with access grants, and we have work in progress. We need to clear those issues and define those patterns, probably follow what Inrupt is doing with access grants. This is all about access delegation which has various prior work. We need to find ways to all work together on this.
* SC: If we can have this topic on the radar, people can be aware that it's happening, at least on Wed, so people can join in. Otherwise we continue as we have been on issues/chat, but I don't find it too active. 
* eP: Tomorrow we'll have discussion with Jackson. He proposed a different AuthN approach for webhook channel type, so we will be able to dive deeper. Hopefully Aaron will join too. 
```